src/js/constants.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
-  consumerKey: '28148-433066734282d2aef4f9aafe',
-  pocket_api_endpoint: 'https://getpocket.com/v3'
+  consumerKey: '38767-70f457996273e97445b7e8f4',
```

Comment:
Changed the consumer key since I had to set up a new app and request access to the Article View API.